libvirt/tests/src/migration/migration_with_disk/migration_with_vhostuser.py

```
@@ -0,0 +1,211 @@
+    def setup_test():
+        """
+        Setup steps before migration
+        """
+        nonlocal vsock_service_id, remote_vsock_service_id, image_path, sock_path
+
+        test.log.info("Setup steps for vhostuser disk migration.")
+
+        sock_path = params.get("source_file", "/tmp/vhost.sock")
+        image_path = data_dir.get_data_dir() + '/test.img'
+        disk_dict = ast.literal_eval(params.get("disk_dict", "{}"))
+        device_target = params.get("target_dev", "vdb")
```

Ruff (0.14.6)

Local variable `device_target` is assigned to but never used

Remove assignment to unused variable `device_target`

(F841)